docs/reference/commands/stage.rst

```diff
@@ -0,0 +1,52 @@
+
+.. _ref_commands_stage:
+
+stage
```

Comment:
I'm wondering what this means to us basically. Not much used for the imagecraft lifecycle.

Reply:
Currently we only do any real work at the build and pack stages. But since this is a craft apps, these commands are available anyway. Moreover, the craft framework could do some magic with some of these commands that users might be interested in, so I kept it. This is more about making sure the experience is consistent with other crafts than only documenting what is really useful to us.